lfs/attribute.go

```diff
@@ -80,15 +80,15 @@ func (a *Attribute) set(key, value string, upgradeables []string, opt InstallOpt
 		if opt.Local {
 			// ignore error for unset, git returns non-zero if missing
 			git.Config.UnsetLocalKey("", key)
-			_, err = git.Config.SetLocal("", key, value)
+			_, err = git.Config.SetLocal("", key, value, opt.Force)
 		} else if opt.System {
 			// ignore error for unset, git returns non-zero if missing
 			git.Config.UnsetSystem(key)
-			_, err = git.Config.SetSystem(key, value)
+			_, err = git.Config.SetSystem(key, value, opt.Force)
 		} else {
 			// ignore error for unset, git returns non-zero if missing
 			git.Config.UnsetGlobal(key)
-			_, err = git.Config.SetGlobal(key, value)
+			_, err = git.Config.SetGlobal(key, value, opt.Force)
```

Comment:
These set the `filter.lfs.{type}` keys, which are not LFS specific. However, I can't imagine what Git would even do with multiple values here.

Reply:
Yeah, the more I think about it the more I think just overwriting is a safer bet for things that Git LFS relies on.